core/emulate.c

```diff
@@ -165,6 +165,14 @@ static const struct em_opcode_t opcode_group3[8] = {
     F(em_neg, op_modrm_rm, op_none, op_none, 0),
 };
 
+static const struct em_opcode_t opcode_group8[8] = {
+    X4(N),
+    F(em_bt, op_modrm_rm, op_simm8, op_none, 0),
```

Comment:
nit: Since we have already specified the same operand types in the opcode table (line 260), there's no need to repeat them here. So this line would become:

`    F(em_bt, op_none, op_none, op_none, 0),`

and the same goes for the next 3 lines. In fact, `opcode_group1` and `opcode_group11` also follow this pattern.

Also nit: Just curious, how are the opcode groups numbered? I haven't been able to make sense of these numbers (1, 3, 8, 11, ...).

Reply:
> Just curious, how are the opcode groups numbered? I haven't been able to make sense of these numbers (1, 3, 8, 11, ...).

This comes from the Intel SDM Vol. 2D, specifically:
- Groups 1, 3 and 11: *Table A-2. One-byte Opcode Map: (00H — F7H)*
- Group 8: *Table A-3. Two-byte Opcode Map: 88H — FFH (First Byte is 0FH)*

Reply:
Wow, thanks! This is the first time I've opened Vol. 2D...